views/partials/_example.njk

```diff
@@ -32,8 +32,11 @@
   <div class="app-example {{ "app-example--tabs" if params.html or params.nunjucks }}">
     <a href="{{ exampleURL }}" class="app-example__new-window" target="_blank">
       Open this
-      <span class="govuk-visually-hidden"> "{{ exampleTitle }}" </span>
-      example in a new window
+      {# Don't use full title visually as the context is shown based on location of this link #}
+      <span class="govuk-visually-hidden">{{ exampleTitle | lower }}</span>
```

Comment:
If we instead removed line 14 (`{% set exampleTitle = exampleTitle + " example" %}`) and added the word 'example' after the `{{ exampleTitle }}` in the iframe title, we could avoid having to use the two different strings here which seems simpler in my mind.

I'm happy with this as it is, but with all the other transformations to `exampleTitle` it feels like this is becoming more complicated than it needs to be. Thoughts?

Reply:
I thought about this and was more worried about the risk of having things get out of sync... Will explore it again before making a decision.

Reply:
Went with your suggestion.